pipeline/software/maya/pipe/rig_publish.py

```diff
@@ -4,15 +4,15 @@
 
 
 def publish(file_name):
-    dir_path = pipe.util.get_rigging_path() / file_name / "RigVersions"
+    dir_path = pipe.util.get_rigging_path() / "Rigs" / file_name / "RigVersions"
     link_dir_path = pipe.util.get_anim_path() / "rigs"
 
     # search directory for all versions and determine new version number
     ls_dir = dir_path.iterdir()
     latest_version = 0
 
     for item in ls_dir:
-        version = int(item.split(".")[-2])
+        version = int(str(item).split(".")[-2])
```

Comment:
Not sure if this is any better than what you have, it feels a little more OO, though. You can change it or not, whatever you prefer, then you should be good to merge this.

```python
version = int(item.suffixes[-2][1:])
```